bigcache.go

```diff
@@ -238,3 +240,67 @@ func (c *BigCache) providedOnRemoveWithMetadata(wrappedEntry []byte, reason Remo
 	shard := c.getShard(hashedKey)
 	c.config.OnRemoveWithMetadata(readKeyFromEntry(wrappedEntry), readEntry(wrappedEntry), shard.getKeyMetadata(hashedKey))
 }
+
+/*
+implement to save slice,resolve append slice no boundary segmentation
+examples:
+	type User struct {
+		Id string `json:"id"`
+		UserName string `json:"user_name"`
+	}
+	var userBs = make([][]byte, 0)
+
+	for i:=1;i<4;i++{
+		var u = &User{
+			Id:i,
+		}
+		bs,_:=json.Marshal(u)
+		userBs = append(userBs,bs)
+	}
+	cache.PutLists("users",userBs...)
+
+	entries,err:=cache.GetLists("users")
+*/
+
+func (c *BigCache) PutLists(key string, entries ...[]byte) (err error) {
```

Comment:
Why do we need that?

Reply:
After appending, the bytes without boundary is obtained